utils/movies.php

```diff
@@ -0,0 +1,49 @@
+<?php 
+require "database.php";
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+// Upload movie function
+
+// example: uploadMovie(1, "title", "asdfg", "p" , 1, "asdfg", 1, 3);
+function uploadMovie($userId, $title, $path, $thumbnailPath, $description, $ageRating, $agerestrictions, $genres){
+    $accepted = getTableRecord("SELECT geverifieerd FROM gebruiker WHERE id = ?", "i", array($userId)   );
+    executeQuery("INSERT INTO film (gebruiker_id, titel, pad, thumbnail_pad, geaccepteerd, beschrijving, kijkwijzer_leeftijd) VALUES (?, ?, ?, ?, ?, ?, ?)", "isssisi", array($userId, $title, $path, $thumbnailPath, $accepted, $description, $ageRating));
```

Comment:
de waardes van deze variabele worden niet gefilterd. Hier zou je 1 van de filter functies voor kunnen pakken in utils/filter.php. Het filteren wordt ook deels gedaan door de functie executeQuery(), maar er wordt bijvoorbeeld niet gecheckt of iets wel echt een cijfer of echt een string is